### DISCUSSION TITLE
Is Volo.Abp.BlazoriseUI compatible with server side blazor?

### DISCUSSION THREAD

**Discussion by wocar** [abpframework/abp#7442] [Q&A] [answered]
Hello I see that 

`Volo.Abp.BlazoriseUI`

has a dependency on:

`Volo.Abp.AspNetCore.Components.WebAssembly`

Is the Abp.BlazoriseUI package compatible with server side blazor?

Thank you!

**Comment by enisn** [comment, marked as answer]
Hello @wocar 

It's not compatible with server-side blazor for now. But we're working on it.

**Reply by 274188A** [reply]
Hi @enisn,
 Is there any update/progress on the availability of a Blazor ServerSide template becoming available?
